library/core/src/alloc/layout.rs

```diff
@@ -520,6 +520,13 @@ impl Layout {
             unsafe { Ok(Layout::from_size_align_unchecked(array_size, align.as_usize())) }
         }
     }
+
+    /// Perma-unstable access to `align` as `Alignment` type.
+    #[unstable(issue = "none", feature = "std_internals")]
```

Comment:
I think we should prevent this from showing up in the documentation, at least while there is no stabilisation planned.
```suggestion
    #[unstable(issue = "none", feature = "std_internals")]
    #[doc(hidden)]
```